javalib/src/main/scala/java/util/Arrays.scala

```diff
@@ -73,18 +75,16 @@ object Arrays {
   @noinline def sort(a: Array[AnyRef], fromIndex: Int, toIndex: Int): Unit =
     sortRangeAnyRefImpl(a, fromIndex, toIndex)
 
-  @noinline
-  def sort[T <: AnyRef](array: Array[T],
-                        comparator: Comparator[_ >: T]): Unit = {
+  @noinline def sort[T <: AnyRef](array: Array[T],
+                                  comparator: Comparator[_ >: T]): Unit = {
```

Comment:
This is a spurious change, as are many other diffs where `@noinline` was put in front on `def` instead of above it. It makes sense for signatures that can stay entirely on one line. But if the signature spans several lines (sometimes *because* of it), then we should not do that, and we should keep `@noinline` above.

Reply:
Thank you for the timely review.

I concur about annotations being on the line above and am changing this PR.

The  `@noinline` on the same line in the sort methods is exactly
the way it appears in the Scala.js source today.  As part of
personal process improvement, I like to figure out how I 
introduce changes which waste reviewer time. I will be
aware of the `annotations above` principal as I continue
to port.